src/MessagingThreadListItem/MessagingThreadListItem.tsx

```diff
@@ -83,38 +87,49 @@ export const MessagingThreadListItem: React.FC<
   }
 
   let indicatorIcon: React.ReactNode;
-  if (hasAlert) {
-    indicatorIcon = alertTooltip ? (
-      <Tooltip
-        content={alertTooltip}
-        placement={Tooltip.Placement.BOTTOM}
-        textAlign={Tooltip.Align.CENTER}
-      >
-        <FontAwesome name="exclamation-triangle" />
-      </Tooltip>
-    ) : (
-      <FontAwesome name="exclamation-triangle" />
+  let indicator: React.ReactNode;
+  if (showNewLabel) {
+    indicator = (
+      <FlexBox className={cssClasses.NEW_LABEL_CONTAINER} justify={Justify.END}>
+        {showNewLabel && <Label color={"success"}>New</Label>}
+      </FlexBox>
     );
-  } else if (!isRead) {
-    indicatorIcon = (
-      <div
-        aria-label={`Unread messages in thread ${title}`}
-        className={classNames(
-          cssClasses.UNREAD_ORB,
-          unreadOrbColor && unreadOrbColorClasses[unreadOrbColor],
-        )}
-      />
+  } else {
+    if (hasAlert) {
+      indicatorIcon = alertTooltip ? (
+        <Tooltip
+          content={alertTooltip}
+          placement={Tooltip.Placement.BOTTOM}
+          textAlign={Tooltip.Align.CENTER}
+        >
+          <FontAwesome name="exclamation-triangle" />
+        </Tooltip>
+      ) : (
+        <FontAwesome name="exclamation-triangle" />
+      );
+    } else if (!isRead) {
+      indicatorIcon = (
+        <div
+          aria-label={`Unread messages in thread ${title}`}
+          className={classNames(
+            cssClasses.UNREAD_ORB,
+            unreadOrbColor && unreadOrbColorClasses[unreadOrbColor],
+          )}
+        />
+      );
+    } else if (status === "active" && hasDraft) {
+      indicatorIcon = <DraftPencilIcon />;
+    }
+    indicator = (
+      <FlexBox className={cssClasses.INDICATOR_CONTAINER} justify={Justify.END}>
+        {indicatorIcon}
+      </FlexBox>
     );
-  } else if (status === "active" && hasDraft) {
-    indicatorIcon = <DraftPencilIcon />;
   }
-  const indicator = (
-    <FlexBox className={cssClasses.INDICATOR_CONTAINER} justify={Justify.END}>
-      {indicatorIcon}
-    </FlexBox>
-  );
 
-  const isIndicatorAlignedWithSubContent = subContent && timestamp;
+  // "New" Label overrides timestamp
+  const showTimestamp = timestamp && !showNewLabel;
```

Comment:
not sure if this is the best way to do this, or if I should just replace `showTimestamp` with `timestamp && !showNewLabel` where I use it later in the code. I thought it was more legible this way though

Reply:
This seems fine to me since you're using it in a couple of places!